cmake/modules/SwiftConfigureSDK.cmake

```diff
@@ -196,8 +206,16 @@ macro(configure_sdk_unix name architectures)
       elseif("${CMAKE_HOST_SYSTEM_NAME}" STREQUAL "Linux")
         set(_swift_android_prebuilt_build "linux-x86_64")
       endif()
-      set(SWIFT_SDK_ANDROID_ARCH_${arch}_NDK_PREBUILT_PATH
+      if("${arch}" STREQUAL "i686")
+        set(SWIFT_SDK_ANDROID_ARCH_${arch}_NDK_PREBUILT_PATH
+          "${SWIFT_ANDROID_NDK_PATH}/toolchains/x86-${SWIFT_ANDROID_NDK_GCC_VERSION}/prebuilt/${_swift_android_prebuilt_build}")
+      elseif("${arch}" STREQUAL "x86_64")
+        set(SWIFT_SDK_ANDROID_ARCH_${arch}_NDK_PREBUILT_PATH
+          "${SWIFT_ANDROID_NDK_PATH}/toolchains/x86_64-${SWIFT_ANDROID_NDK_GCC_VERSION}/prebuilt/${_swift_android_prebuilt_build}")
+      else()
+        set(SWIFT_SDK_ANDROID_ARCH_${arch}_NDK_PREBUILT_PATH
```

Comment:
Can we not use `${SWIFT_SDK_ANDROID_ARCH_${arch}_ALT_SPELLING}-${SWIFT_ANDROID_NDK_GCC_VERSION}` instead?

Reply:
This is what I get:

- `arm-4.9` **NOT** matching $NDK/toolchains value `arm-linux-androideabi-4.9`
- `aarch64-4.9`  **NOT** matching $NDK/toolchains value `arm-linux-androideabi-4.9`
- `i686-4.9`  **NOT** matching $NDK/toolchains value `x86-4.9`
- `x86_64-4.9` **is matching** $NDK/toolchains value `x86_64-4.9`

So, expression `${SWIFT_SDK_ANDROID_ARCH_${arch}_ALT_SPELLING}-${SWIFT_ANDROID_NDK_GCC_VERSION}` not resolving to proper real path in `$NDK/toolchains`.

Details of cmake log:
```
>>>
message(${SWIFT_SDK_ANDROID_ARCH_${arch}_ALT_SPELLING}-${SWIFT_ANDROID_NDK_GCC_VERSION})
arm-4.9
<<<
>>>
message(${SWIFT_SDK_ANDROID_ARCH_${arch}_ALT_SPELLING}-${SWIFT_ANDROID_NDK_GCC_VERSION})
aarch64-4.9
<<<
>>>
message(${SWIFT_SDK_ANDROID_ARCH_${arch}_ALT_SPELLING}-${SWIFT_ANDROID_NDK_GCC_VERSION})
i686-4.9
<<<
>>>
message(${SWIFT_SDK_ANDROID_ARCH_${arch}_ALT_SPELLING}-${SWIFT_ANDROID_NDK_GCC_VERSION})
x86_64-4.9
<<<
```